projects/plugins/jetpack/3rd-party/class.jetpack-amp-support.php

```diff
@@ -177,10 +178,12 @@ public static function disable_likes_admin_bar( $is_admin_bar_button_visible ) {
 	 * @since 6.2.1
 	 */
 	public static function add_stats_pixel() {
-		if ( ! has_action( 'wp_footer', 'stats_footer' ) ) {
+		if ( ! has_action( 'wp_footer', array( 'Automattic\Jetpack\Stats\Tracking_Pixel', 'add_to_footer' ) ) ) {
```

Comment:
You could make this a little clearer if you want by using `::class`.
```suggestion
		if ( ! has_action( 'wp_footer', array( Stats_Tracking_Pixel::class, 'add_to_footer' ) ) ) {
```